### DISCUSSION TITLE
Problems with updating SFTPGo LXC

### DISCUSSION THREAD

**Discussion by tesla-76** [tteck/Proxmox#2989] [Help Needed] [answered]
I tried to run the script to update to version 2.6 but although the prompt at the end states "Updated SFTPGo LXC" the program remains at 2.56.

**Comment by tteck** [comment, marked as answer]
~Nowhere does it say anything about updating~

Edit: It will update when the repo gets updated.

https://github.com/tteck/Proxmox/blob/db157d02e4ee930c3610b3bc58a8c9e73093d150/install/sftpgo-install.sh#L34-L37

**Reply by tesla-76** [reply]
Ah I didn't have the visions! XD
On FTPgo's git page it reports 2.6.0 released about 4 hours ago. Thanks!
